packages/utils/src/lib/auth/siwe/siwe.spec.ts

```
@@ -50,10 +50,23 @@ describe.each([
-      'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhZGRyZXNzIjoiMHgzMTM5QzMzYjcyMTgyMzdCYmQyMjIzNUM3ODA3ODczMTIxNmZEMDViIiwiZXhwIjoxNzI5MjQ1MDA4fQ.Bqe0GYxzww498G0CRYnMfx8x68tmJ_fq59imIZGO54U',
+    const parsedToken = parseJwt<{
+      user_id: string;
+      exp: number;
+    }>(
+      'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiMHhkNTNENGYxMDBBYUJBMzE0YkYwMzNmOTlmODZhMzEyQmZiZERGMTEzIiwiZXhwIjoxNzQxOTQyMzYzfQ.F21W1k_D_IRIns7KacdTkGkrxF5dTjbRA4JQnbmLIHY',
     );
     console.log('parsedToken :', parsedToken);
     expect(parsedToken).toBeTruthy();
   });
+
+  it.only('should generate a valid token from a digest key', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)